packages/react/src/form-field/FormField.css.ts

```diff
@@ -0,0 +1,33 @@
+import { theme } from "../styles";
+import { style } from "../vanilla-extract";
+
+export const description = style({
+  color: theme.colors["fg.default"],
+  fontSize: "12px",
+  lineHeight: "20px",
+  marginTop: "2px",
+});
+
+export const label = style({
+  color: theme.colors["fg.secondary"],
+  fontFamily: theme.fontFamily.sans,
+  fontSize: "14px",
+  marginBottom: "2px",
+});
+
+export const formField = style({
+  display: "flex",
+  flexDirection: "column",
+  maxWidth: theme.maxSize.sm,
+});
+
+export const labelRequired = style({
+  color: theme.colors["border.error"],
+});
+
+export const error = style({
+  color: theme.colors["border.error"],
+  fontSize: "12px",
+  lineHeight: "20px",
+  marginTop: "2px",
+});
```

Comment:
let's not hard code font sizes

for this whole file - since we're using very simple styles we can just replace them all with sprinkles

```suggestion
export const error = sprinkles({
  color: "border.error",
  fontSize: "md"
  mt: "2",
});
```

in fact if we're just using sprinkles and not doing any complicated styles, let's inline the styles into the tsx file rather than creating a separate `.css.ts` file